instance/tests/test_tasks.py

```diff
@@ -72,29 +73,53 @@ def test_mark_active_on_success(self, provisioning_succeeds):
         Test that we when mark_active_on_success=True, the spawn_appserver task will mark the
```

Comment:
I know you didn't comment this, but since we're here can you remove that extraneous 'we'? Thanks :P